docs/diagrams/UiClassDiagram.puml

```diff
@@ -32,26 +28,25 @@ UiManager .left.|> Ui
 UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
-MainWindow *-down-> "1" PersonListPanel
+MainWindow *-down-> "1" ListPanel
 MainWindow *-down-> "1" StatusBarFooter
 MainWindow --> "0..1" HelpWindow
 
-PersonListPanel -down-> "*" PersonCard
+ListPanel -down-> "*" PersonCard
 
 MainWindow -left-|> UiPart
 
 ResultDisplay --|> UiPart
 CommandBox --|> UiPart
-PersonListPanel --|> UiPart
+ListPanel --|> UiPart
 PersonCard --|> UiPart
```

Comment:
I think we can rename the `PersonCard` to `ListCard` in the future. Person is no longer relevant.